procedures/CodeBrowser.ipf

```diff
@@ -117,7 +119,11 @@ Function/S interpretParamType(ptype, paramOrReturn)
 	elseif(ptype & 0x100)
 		typeStr += "dfref"
 	elseif(ptype & 0x200)
-		typeStr += "struct"
+		typeName = "struct"
+		if(FunctionWithWMstruct(funcInfo, typeName))
+			debugPrint("Found WaveMetrics Struct " + typeName)
+		endif
+		typeStr += typeName
```

Comment:
That is rather difficult to grasp as you set `typeName` and then overwrite it.

How about

```
		if(FunctionWithWMstruct(funcInfo, typeName))
                        typeStr += typeName
			debugPrint("Found WaveMetrics Struct " + typeName)
                 else
                         typeStr += "struct"
		endif
```